src/kpro.erl

```diff
@@ -333,14 +360,21 @@ decode(StructName, Bin) when is_atom(StructName) ->
   kpro_structs:decode(StructName, Bin).
 
 decode_message_stream(<<>>, Acc) ->
-  lists:reverse(Acc);
+  Acc;
```

Comment:
because without this change the compressed messages would be reversed twice.
see line 373